plaso/parsers/locate.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""Parser for locate/updatedb database files."""
+
+import os
+
+from dfdatetime import posix_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.lib import dtfabric_helper
+from plaso.lib import errors
+from plaso.lib import specification
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class LocateDatabaseEvent(events.EventData):
+  """Linux Locate Database event data.
+
+  Attributes:
```

Comment:
@sydp I've made some changes but adding comments to let you know what changed.

Docstring for public class attributes should be defined at the class level here and not in `__init__` method.